mantidproject/mantid
# Fixed up some issues in ISIS Powder tutorial

Some small problems were cleared up in the [isis_powder tutorial](http://docs.mantidproject.org/nightly/techniques/ISISPowder-Tutorials.html), and sample config files were added for HRPD.

**To test:**
Build the docs and make sure everything is in order at `docs/html/techniques/ISISPowder-Tutorials.html` and `docs/html/techniques/ISISPowder-HRPD-v1.html#mode`

Fixes #21137 

**Release Notes**
Improvements are only minor, so do not need to be in the release notes

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
